zhaquirks/tuya/ts0601.py

```diff
@@ -83,8 +83,6 @@ class TuyaZemismartSmartCover0601(TuyaWindowCover):
         # <SimpleDescriptor endpoint=1 profile=260 device_type=51 input_clusters=[0, 4, 5, 61184] output_clusters=[25]>
         MODELS_INFO: [
             ("_TZE200_fzo2pocs", "TS0601"),
-            ("_TZE200_zpzndjez", "TS0601"),
-            ("_TZE200_cowvfni3", "TS0601"),
```

Comment:
Way deleting this devices ?

Reply:
my bad , i use a previous reference file ,  those two must be keeped

Reply:
Very OK and not dong some user sad :-))

Reply:
it has been fix and i've noticed that there is 2 references to ("_TZE200_fzo2pocs", "TS0601") device line 85  and 134

Reply:
The device have 2 versions of firmware (on the zigbee module) with different clusters so its have 2 signature entry's that is not unlikely with tuya (most TRVs have 2 different Zigbee modules and have the same ID but is getting different signatures for getting the quirk loaded OK on all devices).